src/views/api_v1.js

```diff
@@ -70,17 +69,17 @@ var Api = {
 		return userID + "_" + global._socket_counter++;
 	},
 
-	getUserID: function (req) {
-		if (!req.user) {
+	getUserID: function (res) {
```

Comment:
This doesn't make sense to be on the response.  This should definitely be request.